agent/src/beerocks/slave/son_slave_thread.cpp

```diff
@@ -3748,8 +3748,10 @@ bool slave_thread::send_cmdu_to_controller(ieee1905_1::CmduMessageTx &cmdu_tx)
         beerocks_header->actionhdr()->radio_mac() = hostap_params.iface_mac;
         beerocks_header->actionhdr()->direction() = beerocks::BEEROCKS_DIRECTION_CONTROLLER;
     }
-    return message_com::send_cmdu(master_socket, cmdu_tx, backhaul_params.controller_bridge_mac,
-                                  backhaul_params.bridge_mac);
+
+    static const std::string MULTICAST_MAC_ADDR = "01:80:c2:00:00:13";
```

Comment:
I guess not since it is defined in the BTL, and the son slave does not link with it.
@morantr - please add this to `network_utils` instead - you should name the variable `MULTICAST_1905_MAC_ADDR` since its the 1905 multicast mac according to the 1905.1 specification.

Reply:
done, although it wouldn't be necessary if we had a unified Agent.
I'm saying it a lot, do I?